### PR TITLE
Create new Department and Designation

### PR DESCRIPTION
**Pull request by EugeneAgyemang**
#44 #

### CODE REVIEW

File: FPIS/Views/AddDepartment.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                );
+
+            if (dialogResult == DialogResult.Yes)
+            {
+                CreateDepartment();
+            }
+            else if (dialogResult == DialogResult.No)
+            {
+                materialTextBoxDepartmentName.Text = "";
+            }
```
Comment (MartyOfMCA) [inline FPIS/Views/AddDepartment.cs]: Please don't delete the entire word the user typed.

Let's say I had to type `Production` and winded up typing `Productio` so I would click No hoping I could add the `n` only to find everything cleared

File: FPIS/Views/AddDesignation.cs [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                );
+
+            if (dialogResult == DialogResult.Yes)
+            {
+                CreateDesignation();
+            }
+            else if (dialogResult == DialogResult.No)
+            {
+                materialTextBoxDesignationName.Text = "";
+            }
```
Comment (MartyOfMCA) [inline FPIS/Views/AddDesignation.cs]: Same issue here as with the Department review

File: FPIS/Views/AddDepartment.cs (hunk 1 above)
Comment (EugeneAgyemang) [reply FPIS/Views/AddDepartment.cs]: Alright,
I've resolved it

File: FPIS/Views/AddDesignation.cs (hunk 2 above)
Comment (EugeneAgyemang) [reply FPIS/Views/AddDesignation.cs]: Resolved as well

**Pull request merged**